includes/class-sensei-share-your-grade.php

```diff
@@ -123,19 +123,19 @@ public function __construct ( $file, $version = '1.0.0' ) {
 		// Set up the data we will need for our output.
 		add_action( 'sensei_course_results_content_inside_after', array( $this, 'setup_course_data_before_output' ), 20 );
 		// TODO - change to use more appropriate hooks when available.
```

Comment:
We can probably remove this comment now 🙂 

Also on the other two instances of this filter.

Reply:
I don't know... I feel like we should always be striving to find a better hook.

j/k removed in bf0585aa7b1f062163c5c36afa675b1ef727cbd6